README.md

```diff
@@ -0,0 +1,31 @@
+# Community DPA Rules for Network Monitor
+This repository contains community DPA rules for network monitor. These rules are examples which often require custom modifications, such as modifying allowed IP-ranges etc. 
+
+# Contributions: 
+The following details are required when adding an example DPA rule
+
+1. /lrl: 
+   * the lrl binary file for the rule
+2. /pcap:  
+    * at least one pcap which would trigger the rule. Additonal pcaps are encouraged as well as pcaps that does not trigger the rule
+    * one yaml file per pcap that describes
+    ```
+    DESCRIPTION:Description of what the rule does and its purpose
```

Comment:
Still needs a space here, then it looks good ... DESCRIPTION:Description -> DESCRIPTION: Description